packages/blade/src/components/BottomSheet/BottomSheet.native.tsx

```diff
@@ -168,6 +170,10 @@ const _BottomSheet = ({
   const renderHandle = React.useCallback((): React.ReactElement => {
```

Comment:
Why was this inside a useCallback? Can't it be just a component as is?

Reply:
Gorhom's render handles and other slots only accepts React.FunctionElement. 

I could've just done `handleComponent={BottomSheetHandle}` but i also need to pass additional props to the components like zIndex, isHeaderEmpty, setHeaderHeight etc thus using a render function. 

I cannot also do this: 
```
            handleComponent={
              <Handle
                isHeaderEmpty={isHeaderEmpty}
                zIndex={zIndex}
                // header={header}
                setHeaderHeight={setHeaderHeight}
              />
            }
```

it will throw "expect a string or class/function but got object"